nemo_skills/dataset/aalcr/prepare.py

```diff
@@ -187,7 +187,7 @@ def write_data_to_file(output_file, data, txt_file_folder, max_context_window, t
                     continue
 
             if n_tokens != entry["input_tokens"]:  # check if the n_tokens exactly match the input_tokens in the entry
-                raise ValueError(f"n_tokens: {n_tokens} != input_tokens: {entry['input_tokens']}")
+                LOG.warning(f"n_tokens: {n_tokens} != input_tokens: {entry['input_tokens']}")
```

Comment:
can you also remove it from the excluded datasets? Then we can run gpu tests to confirm our integration is fixed: https://github.com/NVIDIA-NeMo/Skills/blob/main/tests/gpu-tests/test_eval.py#L47